samples/msal-node-samples/client-credentials-distributed-cache/README.md

```diff
@@ -0,0 +1,89 @@
+# MSAL Node Standalone Sample: Multitenant Daemon App using Client Credentials Grant
+
+This sample demonstrates how to implement an MSAL Node [confidential client application](../../../lib/msal-node/docs/initialize-confidential-client-application.md) to acquire an access token with application permissions using the [OAuth 2.0 Client Credentials Grant](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow).
+
+In addition, this sample uses a custom cache plugin to implement the [distributed token caching](../../../lib/msal-node/docs/caching.md#performance-and-security) pattern. Here, the cache is persisted via [Redis](https://redis.io/) and [node-redis](https://github.com/NodeRedis/node-redis).
+
+## Setup
+
+Locate the folder where `package.json` resides in your terminal. Then type:
+
+```console
+    npm install
+```
+
+## Register
+
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD** service.
+1. Select the **App Registrations** blade on the left, then select **New registration**.
+1. In the **Register an application page** that appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-node-daemon`.
+   - Under **Supported account types**, select **Accounts in any organizational directory (Any Azure AD directory - Multitenant)**.
+   - In the **Redirect URI (optional)** section, select **Public client/native (mobile & desktop)** in the combo-box and enter the following redirect URI: `http://localhost` (this is required for provisioning the app into other tenants via admin consent).
+1. Select **Register** to create the application.
+1. Select **Register** to create the application.
+1. In the app's registration screen, find and note the **Application (client) ID** and **Directory (Tenant) ID**. You use these values in your app's configuration file(s) later.
+1. In the app's registration screen, select the **Certificates & secrets** blade in the left.
+   - In the **Client secrets** section, select **New client secret**.
+   - Type a key description (for instance `app secret`),
+   - Select one of the available key durations (6 months, 12 months or Custom) as per your security posture.
+   - The generated key value will be displayed when you select the **Add** button. Copy and save the generated value for use in later steps.
```

Comment:
You cannot copy the secret later, so this can be added in the notes here "optionally"

Reply:
updated -hope this is more clearer